### PR TITLE
v20.7.4

### PR DESCRIPTION
**Pull request by bmaltais**
* 2023/02/19 (v20.7.4):
    - Add `--use_lion_optimizer` to each training script to use [Lion optimizer](https://github.com/lucidrains/lion-pytorch).
        - Please install Lion optimizer with `pip install lion-pytorch` (it is not in ``requirements.txt`` currently.)
    - Add `--lowram` option to `train_network.py`. Load models to VRAM instead of VRAM (for machines which have bigger VRAM than RAM such as Colab and Kaggle). Thanks to Isotr0py!
        - Default behavior (without lowram) has reverted to the same as before 14 Feb.
    - Fixed git commit hash to be set correctly regardless of the working directory. Thanks to vladmandic!
* 2023/02/15 (v20.7.3):
    - Update upgrade.ps1 script
    - Integrate new kohya sd-script
    - Noise offset is recorded to the metadata. Thanks to space-nuko!
    - Show the moving average loss to prevent loss jumping in `train_network.py` and `train_db.py`. Thanks to shirayu!
    - Add support with multi-gpu trainining for `train_network.py`. Thanks to Isotr0py!
    - Add `--verbose` option for `resize_lora.py`. For details, see [this PR](https://github.com/kohya-ss/sd-scripts/pull/179). Thanks to mgz-dev!
    - Git commit hash is added to the metadata for LoRA. Thanks to space-nuko!
    - Add `--noise_offset` option for each training scripts. 
        - Implementation of https://www.crosslabs.org//blog/diffusion-with-offset-noise 
        - This option may improve ability to generate darker/lighter images. May work with LoRA.

### CODE REVIEW

**Comment by EmmaWebGH** [comment]
Can someone elaborate on how multi GPU works? If I select this when installing, do I need to do anything else, or it's all under the hood?
Have 2x 4090 and would be great to speed up training by running both. Can't find a page in docs here about how this actually functions or what effect this selection has when installing. 
Thanks
